airflow/www/static/js/calendar.js

```diff
@@ -0,0 +1,269 @@
+/* eslint-disable func-names */
+/* eslint-disable no-underscore-dangle */
```

Comment:
We should remove both `eslint-disable`. Neither rule is too cumbersome for this file.

Reply:
I removed the second one, but disabling `func-names` is required to be able to use d3-tip as:
```
      .on('mouseover', function (data) {
        const tt = tipHtml(data);
        dayTip.direction('n');
        dayTip.show(tt, this);
      })
      .on('mouseout', function (data) {
        dayTip.hide(data, this);
      });
```
Using arrow-style functions won't work for some reason.
Removing `func-names` requires extracting the anonymous functions, but that also makes the code a bit heavier. I believe this is the reason `func-names` is currently disabled in `tree.js` as well.

What do you think? should we keep it this way or should we extract the tooltip functions?

Reply:
I’m far from fluent in JavaScript, but would sound like this pass?

```js
.on('mouseout, function hideDayTipOnMouseOut(data) {
  dayTip.hide(data, this)
});
```

Reply:
@uranusjr thanks, `eslint` seems to be ok with that, I have updated the code and removed the eslint exception :+1: